dotnet/src/Connectors/Connectors.OpenAI/TextToAudio/AzureOpenAITextToAudioService.cs

```diff
@@ -0,0 +1,161 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Contents;
+using Microsoft.SemanticKernel.Http;
+using Microsoft.SemanticKernel.Services;
+using Microsoft.SemanticKernel.TextToAudio;
+
+namespace Microsoft.SemanticKernel.Connectors.OpenAI;
+
+/// <summary>
+/// Azure OpenAI text-to-audio service.
+/// </summary>
+[Experimental("SKEXP0005")]
+public sealed class AzureOpenAITextToAudioService : ITextToAudioService
+{
+    private readonly Dictionary<string, object?> _attributes = new();
+    private readonly ILogger _logger;
+    private readonly HttpClient _httpClient;
```

Comment:
The pattern we want to promote is having a seperate client to encapsulate all of the Http logic. Ideally we will get an official client for this service which we can use in the future.

I'm OK with this change happening in another PR, if that is the plan please link the issue and close this comment.

Reply:
> Ideally we will get an official client for this service which we can use in the future.

That's exactly the reason why I placed all the logic together, I'm expecting to use official client for this functionality in the future. But we don't know when it's going to happen, so I moved all HTTP-related logic to separate client in this PR. Thanks for feedback!